packages/node-patches/test/fs/lstat.ts

```diff
@@ -38,7 +38,7 @@ describe('testing lstat', () => {
 
           const patchedFs = Object.assign({}, fs);
           patchedFs.promises = Object.assign({}, fs.promises);
-          patcher(patchedFs, [path.join(fixturesDir)]);
+          patcher(patchedFs, [path.join(fixturesDir)], []);
```

Comment:
nit: would be nice to make a `const NO_GUARDS = []` so the meaning of the arguments is more obvious.

Alternatively maybe the function wants to have a default for this new argument so you don't need all these updates in the test

Reply:
Good idea. Will update.

Reply:
Ended up not going with an API change on that function .